setup.py

```diff
@@ -516,6 +517,10 @@ def pxd(name):
     '_libs.lib': {
         'pyxfile': '_libs/lib',
         'depends': lib_depends + tseries_depends},
```

Comment:
need to add missing to lib as well

Reply:
Can do.  At the moment we've basically only implemented the pxfiles "rule" for tslibs.  Should we go through and fill them out for the others?

In case I haven't mentioned it recently: I'm _really_ looking forward to using cythonize and not worrying about this.